ui/app/components/auth/form/base.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+
+/**
+ * @module Auth::Base
+ *
+ * @example
+ *
+ * @param {string} param -
+ */
+
+export default class AuthBase extends Component {
+  @service auth;
+
+  maybeMask = (field) => {
+    if (field === 'token' || field === 'password') {
+      return 'password';
+    }
+    return 'text';
+  };
```

Comment:
This feels like it would eventually be great as a helper!

Reply:
Yeah! It's just a local helper now. I'm not sure where else this would be useful, since `FormField` (or whatever 2.0 is) will likely want to offload this distinction to the parent so it can be customized. Though it could possible be useful there 🤔 Did you have somewhere else in mind?

Reply:
Coming back to this - I decided to keep this local since universally we actually _don't _ want input types to be `password` since password managers get confused, so this is exclusive to the login form. 

Thanks for the suggestion! It was useful to think through